tiledb/bioimg/converters/base.py

```diff
@@ -362,23 +432,99 @@ def _convert_level_to_tiledb(
             if chunked or max_workers:
                 inv_axes_mapper = axes_mapper.inverse
 
-                def tile_to_tiledb(level_tile: Tuple[slice, ...]) -> None:
+                def tile_to_tiledb(
+                    level_tile: Tuple[slice, ...]
+                ) -> Tuple[np.ndarray, ...]:
                     source_tile = inv_axes_mapper.map_tile(level_tile)
                     image = reader.level_image(level, source_tile)
                     out_array[level_tile] = axes_mapper.map_array(image)
 
+                    return np.amin(image, axis=min_max_indices), np.amax(
+                        image, axis=min_max_indices
+                    )
+
                 ex = ThreadPoolExecutor(max_workers) if max_workers else None
                 mapper = getattr(ex, "map", map)
-                for _ in tqdm(
+                for tile_min, tile_max in tqdm(
                     mapper(tile_to_tiledb, iter_tiles(out_array.domain)),
                     desc=f"Ingesting level {level}",
                     total=num_tiles(out_array.domain),
                     unit="tiles",
                 ):
+                    compute_channel_minmax(channel_min_max, tile_min, tile_max)
                     pass
                 if ex:
                     ex.shutdown()
             else:
                 image = reader.level_image(level)
                 out_array[:] = axes_mapper.map_array(image)
-    return uri
+
+                compute_channel_minmax(
+                    channel_min_max,
+                    np.amin(image, axis=min_max_indices),
+                    np.amax(image, axis=min_max_indices),
+                )
+
+    level_metadata["Uri"] = uri
```

Comment:
idk if its only me but do we need to use `camel-case`  for that? You can easily skip though.